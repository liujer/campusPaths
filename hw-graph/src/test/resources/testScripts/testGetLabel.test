# Tests the method getLabel in DLGraph by adding nodes
# and edges to a graph and checking if the label returned
# is correct
CreateGraph B
AddNode B one
AddNode B two
AddNode B three
AddEdge B one two e12
AddEdge B three two e32
GetLabel B one two
GetLabel B three two
