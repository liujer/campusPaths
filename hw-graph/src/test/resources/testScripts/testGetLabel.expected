Created graph B
Added node one to B
Added node two to B
Added node three to B
Added edge e12 to B
Added edge e32 to B
Label of one and two in B output: e12
Label of three and two in B output: e32